### PR TITLE
Adds ES Class status item

### PR DESCRIPTION
**Pull request by pzuraq**
cc @locks @rwjblue @wycats

### CODE REVIEW

File: app/components/roadmap-page/-utils/features.yaml [hunk 1]
```diff
@@ -287,7 +287,64 @@ features:
...
+
+      As it turns out, the existing object model is already highly cross-compatible
+      with the new class syntax. The [ES Class
+      RFC](https://github.com/emberjs/rfcs/blob/master/text/0240-es-classes.md)
+      outlined a path forward that required few changes to the existing object model,
+      mostly around fixing broken features such as observers and event listeners,
+      and allowing "zebra-striping" - being able to go back and forth between native
+      classes and EmberObject based classes using the old style.
+
+      The RFC also touched on decorators, but didn't specify any specify any to be
```
Comment (acorncom) [inline app/components/roadmap-page/-utils/features.yaml]: duplicate `specify any` here?

Comment (pzuraq) [reply app/components/roadmap-page/-utils/features.yaml]: that would be correct, will update

**Comment by SergeAstapov** [comment]
@pzuraq FYI this needs to be updated as features were split into separate yaml files

**Comment by pzuraq** [comment]
Updated!

**Pull request merged**